ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -359,51 +373,54 @@ tests:
     workflow: openshift-e2e-openstack-ipi
 - as: e2e-openstack-proxy
   optional: true
+  skip_if_only_changed: (^(docs|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(alibabacloud|aws|azure|baremetal|gcp|ibmcloud|libvirt|ovirt|vsphere)/)
   steps:
     cluster_profile: openstack-vh-mecha-central
     workflow: openshift-e2e-openstack-proxy
 - as: openstack-manifests
-  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(aws|azure|baremetal|gcp|libvirt|ovirt|vsphere)/)
+  skip_if_only_changed: (^(docs|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(alibabacloud|aws|azure|baremetal|gcp|ibmcloud|libvirt|ovirt|vsphere)/)
   steps:
     cluster_profile: openstack-vexxhost
     workflow: openshift-manifests-openstack
 - as: e2e-vsphere
   optional: true
-  run_if_changed: (^(data/.*|pkg.*)/vsphere)|(^vendor/github.com/vmware/)
+  skip_if_only_changed: (^(docs|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(alibabacloud|aws|azure|baremetal|gcp|ibmcloud|libvirt|openstack|ovirt)/)
```

Comment:
This changes the job from one that runs only on vSphere specific changes to one that runs on all changes except those specific to a non-vSphere platform. I think that's probably also out of scope.

Reply:
I'm thinking there may be times where code common to all is changed and we would want to run vsphere just to test if it works